app/controllers/articles_controller.rb

```diff
@@ -1,31 +1,36 @@
 class ArticlesController < ApplicationController
-    def new
-        @pg_title = "Create Post | Haxxor News"
-        @article = Article.new
-    end
-    
-    def create
-        @article = Article.new(article_params)
-        
-        if @article.save
-            redirect_to :controller => 'home', :action => 'index'
-        else
-            render 'new'
-        end
-    end
-    
-    def show
-        @article = Article.find(params[:id])
-    end
-    
-    def destroy
-        @article = Article.find(params[:id])
-        @article.destroy
-        redirect_to :controller => 'home', :action => 'index'
-    end
-    
-    private
-    def article_params
-        params.require(:article).permit(:title, :body)
+  def new
+    @pg_title = "Create Post | Haxxor News"
```

Comment:
would recommend naming this variable `@page_title`. `pg` is the shorthand name for postgres so this is a bit confusing at first glance.